package.json

```diff
@@ -39,7 +39,9 @@
     "grunt-less-to-sass": "duvillierA/grunt-less-to-sass#0.0.11",
     "grunt-sass": "^3.1.0",
     "jit-grunt": "^0.10.0",
-    "node-sass": "^7.0.1"
+    "node-sass": "^7.0.1",
+    "simple-keyboard": "^3.6.0",
```

Comment:
I believe that the simple-keyboard and simple-keyboard-layouts dependencies should be moved under devDependencies. Since we commit the Javascript/CSS vendor files to the main project, we only need to load them from NPM in order to run the Phing `copynodemodules` task -- and that is a development activity rather than a regular day-to-day operation.  (I realize that there is some existing inconsistency here, and I believe there has been discussion about moving the autocomplete library down in a different PR... just hoping we can avoid compounding the problem here).

Reply:
I just put it in there because autocomplete.js was also there. But your explanation makes sense. Moved it to devDependencies.